ansible/roles/httpd/tasks/configure.yml

```diff
@@ -1,3 +1,6 @@
 ---
-  
-# TODO: Add httpd configuration files for the application. (with j2 templates)
\ No newline at end of file
+
+# TODO: Add httpd configuration files for the application. (with j2 templates)
+- template: src=../templates/etc/httpd/conf.d/access.conf.j2 dest=/etc/httpd/conf.d/access.conf owner=apache group=apache mode=0644
+  tags:
+    -httpd configuration
```

Comment:
Tag should be one word, delimited with underscore (_). So use `httpd_configuration` instead.